buf.yaml

```diff
@@ -0,0 +1,27 @@
+# buf.yaml files should always be in the root of your repository.
+build:
+  roots:
+    # roots is the directories that all .proto files are contained within, and
+    # all imports must derive from.
+    # Only .proto files within these directory will be compiled.
+    - model
+  # excludes is the list of directories within root to exclude.
+  excludes:
+    - model/build
+
+# lint contains the options for lint checks.
+lint:
+  # use is the list of checker categories and ids to use for buf check lint.
+  use:
+    # The BASIC category includes everything from the MINIMAL category, as well as the STYLE_BASIC category.
```

Comment:
Is there a link where the categories are explained? If so it would be good to add it to the comment. Also just a reminder to add whatever this linter is checking into the style guide under a 'protobuf' section

Reply:
https://buf.build/docs/lint-checkers, category details are here. Ok, I will add these details in the documentation as well and other categories brief intro in the comments.

Reply:
Can you also add a link here in code for quick reference for readers?

Reply:
Done

Reply:
@anandwana001 Nit: `Three category` should be `Three categories`

Reply:
ok